tests/helpers/providers/record_provider.py

```diff
@@ -51,7 +51,23 @@ def author_record():
             "_collections": ["Authors"],
         }
 
-    def record(self, record_type, data=None, with_control_number=False):
+    @staticmethod
+    def add_citations(citation_records=[]):
```

Comment:
misleading optional value, it does nothing and useless if nothing passed, maybe it's better to just fail.

Reply:
something like would be better `get_list_of_refereces_from_record_ids`